runtime/terms_test.go

```diff
@@ -71,3 +71,45 @@ func TestTermsTypes(t *testing.T) {
 		}
 	}
 }
+
+func TestTermsCounter(t *testing.T) {
+	tests := []struct {
+		failpointTerm    string
+		runAfterEnabling int
+		wantCount        int
+	}{
+		{
+			failpointTerm:    `10*sleep(10)->1*return("abc")`,
+			runAfterEnabling: 12,
+			// This example tests mods which allows users to restrict the
+			// number of failpoint actions as against their callsite executions.
+			// This is the reason why wantCount < runAfterEnabling
+			// In a real world example you can hit a code spot a million times but
+			// using mods restrict the associated fallpoint actions to run twice.
```

Comment:
It's a good test case, but the comment isn't clear to me.
```suggestion
			// Note the chain of terms is allowed to be executed 11 time at most, 
			// including 10 times for the first term `10*sleep(10)` and 1 time for
			// the second term `1*return("abc")`. So it's only evaluated 11 times
			// even it's triggered 12 times.
```

Reply:
Done. Thanks for your suggestions!